code/modules/projectiles/guns/energy/pulse.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/energy/pulse
 	name = "pulse rifle"
-	desc = "A heavy-duty, multifaceted energy rifle with three modes. Preferred by front-line combat personnel."
+	desc = "The NT-PR2. A heavy-duty, multifaceted energy infantry rifle that can stun, kill and DESTROY. Considered a great weapon by front-line fighters."
```

Comment:
I think i preferred the old description here

Reply:
Agreed